itest/litd_custom_channels_test.go

```diff
@@ -2098,7 +2086,7 @@ func testCustomChannelsLiquidityEdgeCases(ctx context.Context,
 
 	// Now Fabia creates the normal invoice.
 	invoiceResp = createAssetInvoice(
-		t.t, erin, fabia, 10_000, assetID,
+		t.t, erin, fabia, 0, assetID, withMsatAmount(170_000_000),
```

Comment:
Should we add a new case for this new functionality? Or at least update the comment above this to point out that we're testing a specific case?